src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -30,8 +30,12 @@ module sonic-system-aaa {
                 }
 
                 leaf login {
-                    type string;
-                    description "AAA authentication/authorization/accounting methods - local/tacacs+/disable";
+                    type string {
+                        pattern 'radius|tacacs\+|local|default' {
```

Comment:
Can you please add config into https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json ?

Reply:
Please also add configuration schema here https://github.com/Azure/sonic-swss/blob/53c630b82d429db892bf288dd9f323c5a8370cd5/doc/Configuration.md

Reply:
AAA setting already in sample_config_db.json:

        "AAA": {
            "authentication": {
                "login": "local"
            },
            "authorization": {
                "login": "local"
            },
            "accounting": {
                "login": "local"
            }
        },

Reply:
Will update Configuration.md in another PR because it's in different repo.

Reply:
Created a new PR for update configuration.md in sonic-swss repo: https://github.com/Azure/sonic-swss/pull/2168